gui/default/index.html

```diff
@@ -768,6 +768,10 @@ <h4 class="panel-title">
                       <th><span class="fas fa-fw fa-folder"></span>&nbsp;<span translate>Folders</span></th>
                       <td class="text-right" ng-attr-title="{{deviceFolders(deviceCfg).map(folderLabel).join(', ')}}">{{deviceFolders(deviceCfg).map(folderLabel).join(", ")}}</td>
                     </tr>
+                    <tr ng-if="webAddress[deviceCfg.deviceID.substr(0, 6)]">
+                      <th><span class="fas fa-fw fa-link"></span>&nbsp;<span translate>web address</span></th>
+                      <td class="text-right" ng-attr-title="{{webAddress[deviceCfg.deviceID.substr(0, 6)]}}"><a ng-click="goToRemoteDevice(deviceCfg.deviceID.substr(0, 6))">{{webAddress[deviceCfg.deviceID.substr(0, 6)]}}</a></td>
```

Comment:
This should ideally be a more regular angular `<a href="{{something}}">` element, so that the user can for example right click it to open in a separate tab and stuff like that.